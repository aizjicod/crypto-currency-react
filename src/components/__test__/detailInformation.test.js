import { render } from '@testing-library/react';
import React from 'react';
import DetailInformation from '../detailinformation';

test('tesing snapshot for DetailInformation component', () => {
  const component = render(
    <DetailInformation
      img="https://assets.coingecko.com/coins/images/1/thumb/bitcoin.png?1547033579"
      name="bitcoin"
      totalSupply={200}
      circulatingSupply={20}
      priceChange1yr={-50}
      lastTickers={5}
    />,
  );
  expect(component).toMatchSnapshot();
});

test('tesing second snapshot for DetailInformation component', () => {
  const component = render(
    <DetailInformation
      img="https://assets.coingecko.com/coins/images/1/thumb/bitcoin.png?1547033579"
      name="bitcoin"
      totalSupply={1111111000000}
      circulatingSupply={135464685}
      priceChange1yr={+1354200687651}
      lastTickers={3857351203571}
    />,
  );
  expect(component).toMatchSnapshot();
});
