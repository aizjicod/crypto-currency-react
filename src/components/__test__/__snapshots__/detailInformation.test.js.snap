// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tesing second snapshot for DetailInformation component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="header-detail"
      >
        <img
          alt="bitcoin"
          src="https://assets.coingecko.com/coins/images/1/thumb/bitcoin.png?1547033579"
        />
        <h1>
          bitcoin
        </h1>
      </header>
      <h2
        class="detail-h2"
      >
        Principal Data
      </h2>
      <div
        class="data-div"
      >
        <p
          class="data-p"
        >
          Total supply: 1111111000000
        </p>
        <p
          class="data-p"
        >
          circulating supply: 135464685
        </p>
        <p
          class="data-p green"
        >
          change of coin in a year: 1354200687651%
        </p>
        <p
          class="data-p"
        >
          last tickets: 3857351203571
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <header
      class="header-detail"
    >
      <img
        alt="bitcoin"
        src="https://assets.coingecko.com/coins/images/1/thumb/bitcoin.png?1547033579"
      />
      <h1>
        bitcoin
      </h1>
    </header>
    <h2
      class="detail-h2"
    >
      Principal Data
    </h2>
    <div
      class="data-div"
    >
      <p
        class="data-p"
      >
        Total supply: 1111111000000
      </p>
      <p
        class="data-p"
      >
        circulating supply: 135464685
      </p>
      <p
        class="data-p green"
      >
        change of coin in a year: 1354200687651%
      </p>
      <p
        class="data-p"
      >
        last tickets: 3857351203571
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`tesing snapshot for DetailInformation component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="header-detail"
      >
        <img
          alt="bitcoin"
          src="https://assets.coingecko.com/coins/images/1/thumb/bitcoin.png?1547033579"
        />
        <h1>
          bitcoin
        </h1>
      </header>
      <h2
        class="detail-h2"
      >
        Principal Data
      </h2>
      <div
        class="data-div"
      >
        <p
          class="data-p"
        >
          Total supply: 200
        </p>
        <p
          class="data-p"
        >
          circulating supply: 20
        </p>
        <p
          class="data-p red"
        >
          change of coin in a year: -50%
        </p>
        <p
          class="data-p"
        >
          last tickets: 5
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <header
      class="header-detail"
    >
      <img
        alt="bitcoin"
        src="https://assets.coingecko.com/coins/images/1/thumb/bitcoin.png?1547033579"
      />
      <h1>
        bitcoin
      </h1>
    </header>
    <h2
      class="detail-h2"
    >
      Principal Data
    </h2>
    <div
      class="data-div"
    >
      <p
        class="data-p"
      >
        Total supply: 200
      </p>
      <p
        class="data-p"
      >
        circulating supply: 20
      </p>
      <p
        class="data-p red"
      >
        change of coin in a year: -50%
      </p>
      <p
        class="data-p"
      >
        last tickets: 5
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
